src/components/10-atoms/icon/index.d.ts

```diff
@@ -1,23 +1,3 @@
-/**
- * Either one of enumerated string literals, or:
- *
- * - a string containing ".svg", will be loaded as URL
- * - a string containing "<svg", will be inlined
- */
-export type Icon =
-  | 'add'
-  | 'arrow-left'
-  | 'arrow-right'
-  | 'collapse'
-  | 'document'
-  | 'download'
-  | 'email'
-  | 'expand'
-  | 'mobile'
-  | 'phone'
-  | 'search'
-  | 'upload'
-  | 'cloud-upload'
-  | 'axa-logo'
-  | 'axa-logo-open'
-  | String;
+export type Icon = String;
```

Comment:
I'm against removing the possible values. This is very valuable when you have autocomplete in your IDE, as you see directly what you can choose from.